src/console_cmds.cpp

```diff
@@ -1338,7 +1338,17 @@ static bool ConNewGame([[maybe_unused]] uint8_t argc, [[maybe_unused]] char *arg
 		return true;
 	}
 
-	StartNewGameWithoutGUI((argc == 2) ? std::strtoul(argv[1], nullptr, 10) : GENERATE_NEW_SEED);
+	uint32_t seed = GENERATE_NEW_SEED;
+	if (argc > 2) {
+		auto param = ParseInteger(argv[1]);
+		if (!param.has_value()) {
+			IConsolePrint(CC_ERROR, "The given seed must be a valid number.");
+			return true;
```

Comment:
Doesn't the return value indicate failure?
```suggestion
			return false;
```

Reply:
There are many cases where a print with `CC_ERROR` results in `return true`.
What is, currently, functionally does show the help text for the command when `false` is returned.

In many cases there is, for me at least, no real need for the complete help text to be shown upon an error. Especially as the help text will be shown below the error, making the error harder to spot.